### ISSUE TITLE
Error in lib/combotreebox.py

### ISSUE THREAD

**Issue by ghost** [wxWidgets/Phoenix#965]
I am found error in 
[lib/combotreebox.py](https://github.com/wxWidgets/Phoenix/blob/master/wx/lib/combotreebox.py#L605)
in function GetString(self, item)
_item_ needs check, because some times it type is _int_ instead of _wx.TreeItemId_
it checked with
```python
if item:
```
but correct check is
```python
if isinstance(item, wx.TreeItemId):
```

**Comment by RobinD42** [comment]
In what cases is it an int?

**Comment by NeiroNx** [comment]
comboteebox used as in example:
```python
        self.mChoice2List = {"root1":["item1","item2", ...],{"root2":["item3","item4", ...], ...}
        self.choice3 = ComboTreeBox(self, wx.NewId(), size=(400,-1),style = wx.CB_READONLY)
        self.choice3.SetFont(font9)
        for nItem in self.mChoice2List:
            item = self.choice3.Append(nItem)
            for pItem in self.mChoice2List[nItem]:
                self.choice3.Append(pItem, item)
        self.choice3.SetValue("item1")
```
it used in a big program, and i am can not get full stack trace about it. 
`Traceback (most recent call last):
  File "C:\Program Files\Python37\lib\site-packages\wx\lib\combotreebox.py", line 606, in GetString
    return self._tree.GetItemText(item)
TypeError: TreeCtrl.GetItemText(): argument 1 has unexpected type 'int'`
and it repeat for each element at combo populate

**Comment by RobinD42** [comment]
Where is the code that is calling `GetString`? Where does it get the int that it is passing to `GetString`?

**Comment by NeiroNx** [comment]
I am do not know. may be it code in lib\combotreebox.py
i am only use `self.choice3.GetValue()` to get result. But error appeaar many times - i think on each element on connecting combotbox items with tree items.
Check incoming type in GetString is simle way for me. I am have no time to analyse full library code.

**Comment by RobinD42** [comment]
Please create a small runnable sample that demonstrates the problem. I haven't been able to reproduce it.

**Comment by RobinD42** [comment]
Actually, if you are using Python3 then this may be related to missing a `__bool__` method, and have been fixed by 1fa390846656da6b554f4c37f13d1c5f239e24f5. Please test with a new build from the snapshots folder and see if the exception still happens.

**Comment by infinity77** [comment]
@swt2c : no response, no sample app. As you wish, but I’d just close this.
